rca/static_src/sass/abstracts/_variables.scss

```diff
@@ -40,18 +40,75 @@ $color--meta-lightbg: $color--dark-grey;
 
 $color--focus: $color--blue;
 
+// --------------------------------- Font faces -------------------------------------
+
+/* source-sans-3-latin-400-normal */
+@font-face {
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-display: swap;
+    font-weight: 400;
+    src: url(https://cdn.jsdelivr.net/fontsource/fonts/source-sans-3@latest/latin-400-normal.woff2)
+            format('woff2'),
+        url(https://cdn.jsdelivr.net/fontsource/fonts/source-sans-3@latest/latin-400-normal.woff)
+            format('woff');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
```

Comment:
Interesting! I note that in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range):
> If the page doesn't use any character in this range, the font is not downloaded; if it uses at least one, the whole font is downloaded.

Does this mean that the font will basically always be downloaded in its entirety? Perhaps I'm misunderstanding, but I'm struggling to understand the performance benefit.